From c292bab2ba04dbc8e4c62a6f52b91c4e983f057e Mon Sep 17 00:00:00 2001
From: Huang Chaofan <chaofan.huang@nxp.com>
Date: Fri, 8 Jun 2018 10:38:24 +0800
Subject: [PATCH 3928/5242] MLK-18545: [Linux_8QXP_MEK]Video:Video disappeared
 after long time loop play the test clip, 100%

commit  584ed53f007b535864794364f4caa936f2928e57 from
https://source.codeaurora.org/external/imx/linux-imx.git

fix it by comment out resolution change event in vpu decoder driver

Signed-off-by: Huang Chaofan <chaofan.huang@nxp.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 drivers/mxc/vpu-decoder-b0/vpu_b0.c |    2 ++
 1 file changed, 2 insertions(+)

diff --git a/drivers/mxc/vpu-decoder-b0/vpu_b0.c b/drivers/mxc/vpu-decoder-b0/vpu_b0.c
index 084235b..adf474e 100644
--- a/drivers/mxc/vpu-decoder-b0/vpu_b0.c
+++ b/drivers/mxc/vpu-decoder-b0/vpu_b0.c
@@ -1763,6 +1763,7 @@ static void vpu_api_event_handler(struct vpu_ctx *ctx, u_int32 uStrIdx, u_int32
 		}
 		break;
 	case VID_API_EVENT_RES_CHANGE:
+#if 0
 		{
 			const struct v4l2_event ev = {
 			.type = V4L2_EVENT_SOURCE_CHANGE,
@@ -1770,6 +1771,7 @@ static void vpu_api_event_handler(struct vpu_ctx *ctx, u_int32 uStrIdx, u_int32
 		};
 		v4l2_event_queue_fh(&ctx->fh, &ev);
 		}
+#endif
 		break;
 	case VID_API_EVENT_STR_BUF_RST: {
 		pSTREAM_BUFFER_DESCRIPTOR_TYPE pStrBufDesc;
-- 
1.7.9.5

